user-service/src/main/java/hcmus/alumni/userservice/utils/ImageUtils.java

```diff
@@ -136,5 +137,13 @@ public static boolean isPng(MultipartFile imageFile) {
 	public String getAvatarPath() {
 		return avatarPath;
 	}
+	
+	public String getAvatarPath(String id) {
+		return this.avatarPath + id + "/";
+	}
+	
+	public String getCoverPath(String id) {
+		return this.coverPath + id + "/";
+	}
```

Comment:
```suggestion
	public String getAvatarPath() {
		return this.avatarPath;
	}
	
	public String getCoverPath() {
		return this.coverPath;
	}
```
tên file sẽ là userId lun như cái cmt ở api cover và avatar